assets/scss/components/_external-markup.scss

```diff
@@ -0,0 +1,27 @@
+/*
+External Markup
+
+Partial HTML fragments from other services may be vanilla HTML with no classes upon which to apply styles.
+This module can house any wrapper styles to apply to this content.
+
+Markup:
+<div class="{{modifier_class}}">
+  <h2>Heading level 2</h2>
+  <p>Paragraph text</p>
+  <a href="">Anchor</a>
+</div>
+
+.external-markup--xsmall - for extra small external markup
+
```

Comment:
if there is only one class in the component, do you need to use the modifier class here?